relay-replays/src/snapshots/relay_replays__recording__tests__scrub_pii_key_based_edge_cases.snap

```diff
@@ -0,0 +1,45 @@
+---
+source: relay-replays/src/recording.rs
+expression: scrubbed
+---
+[
+  {
+    "data": {
+      "payload": {
+        "data": {
+          "method": "POST [ip]",
+          "request": {
+            "body": {
+              "api_key": "[Filtered]",
```

Comment:
`api_key` is scrubbed because `data.payload.data` is in `PII_FIELDS`, and `api_key` is in [`PASSWORD_KEY_REGEX`](https://github.com/getsentry/relay/blob/258783fc348c7920cd3651734688ba95d12f5968/relay-general/src/pii/regexes.rs#L275-L279).